lang/c++/README

```diff
@@ -39,8 +39,8 @@ mkdir build
 cd build
 cmake -G "Unix Makefiles" ..
 
-If it doesn't work, either you are missing some packages (boost, flex or bison),
-or you need to help configure locate them.
+If it doesn't work, either you are missing boost package or you need to help
+configure locate it.
```

Comment:
So, `flex` and `bison` are no more needed ? (I am not familiar with the C++ SDK!)

Reply:
Yes. This dependency was removed 12 years ago :D
https://issues.apache.org/jira/browse/AVRO-956

Reply:
(sorry for the laugh, I was just very surprised that these obsolete files have been here for so long)